docs/articles/documentation/reference/testcafe-api/runner/run.md

```diff
@@ -55,28 +55,22 @@ After all tests are finished, call the [testcafe.close](../testcafe/close.md) fu
 
 ```js
 const createTestCafe = require('testcafe');
-let testcafe         = null;
-
-createTestCafe('localhost', 1337, 1338)
-    .then(tc => {
-        testcafe     = tc;
-        const runner = testcafe.createRunner();
-
-        return runner.run({
-            skipJsErrors: true,
-            quarantineMode: true,
-            selectorTimeout: 50000,
-            assertionTimeout: 7000,
-            pageLoadTimeout: 8000,
-            speed: 0.1,
-            stopOnFirstFail: true
-        });
-    })
-    .then(failed => {
-        console.log('Tests failed: ' + failed);
-        testcafe.close();
-    })
-    .catch(error => { /* ... */ });
```

Comment:
@AndreyBelym Do you think we caught on purpose here?
I'm not sure it's really important that we `try...catch` here now with `await`

Reply:
Yes, we can skip `catch` block with `await`. But now I see that it's better to move `testcafe.close` into an `finally` block:
```js
const testCafe = await createTestCafe();

try {
  const runner = testCafe.createRunner();
  //...
}
finally {
    await testCafe.close();
}
```

Reply:
I'm not sure if we should use this pattern in all examples since it is a bit verbose. We had some users that didn't use `testcafe.close` at all.